macros/is_databricks_sql_warehouse.sql

```diff
@@ -0,0 +1,15 @@
+{% macro is_databricks_sql_warehouse(target) %}
```

Comment:
we may want to add this to fivetran_utils in the future since Databricks SQL Warehouse folks using other packages with incremental models will have the same issue

Reply:
I agree. This is something we will migrate there in the future.